capstone/src/main/java/com/google/sps/AssociationResult.java

```diff
@@ -82,6 +82,10 @@ public float getWeight() {
     return key;
   }
 
+  public String getNormalizedString() {
+    return content.toLowerCase().trim();
```

Comment:
I recall that upper and lower cases interfered with your positive and negative associations. Is there a reason why you chose to go with lowercase?

Reply:
It doesn't really matter as long as its all the same. I just wanted to get rid of cases where two things were identified as different because they had different cases. The choice between lowercase and uppercase here is arbitrary as it is only used to standardize comparison.